src/Adapter/MSTest.CoreAdapter/TestMethodFilter.cs

```diff
@@ -96,6 +96,19 @@ internal object PropertyValueProvider(TestCase currentTest, string propertyName)
             return null;
         }
 
+        private static object SafeInvoke<T>(Func<T> action)
+        {
+            try
+            {
+                return action.Invoke();
+            }
+            catch (Exception)
+            {
+            }
```

Comment:
Needs logging.

Reply:
added logger, though the warning would be strange